File: bannerlayout/src/main/java/com/wkz/bannerlayout/imagemanager/PicassoImageManager.java
package com.wkz.bannerlayout.imagemanager;

import android.support.annotation.NonNull;
import android.view.ViewGroup;
import android.widget.ImageView;

import com.squareup.picasso.Picasso;
import com.wkz.bannerlayout.listener.BannerModelCallBack;
import com.wkz.bannerlayout.listener.ImageDisplayManager;
import com.wkz.bannerlayout.widget.BannerDefaults;

public class PicassoImageManager implements ImageDisplayManager {
    @NonNull
    public ImageView display(@NonNull ViewGroup container, @NonNull BannerModelCallBack model) {
        ImageView imageView = new ImageView(container.getContext());
        Picasso.get()
                .load((String) model.getBannerUrl())
                .placeholder(BannerDefaults.GLIDE_PLACEHOLDER_DRAWABLE)
                .error(BannerDefaults.GLIDE_ERROR_DRAWABLE)
                .into(imageView);
        return imageView;
    }
}

File: bannerlayout/src/main/java/com/fphoenixcorneae/bannerlayout/imagemanager/PicassoImageManager.kt
package com.fphoenixcorneae.bannerlayout.imagemanager

import android.view.ViewGroup
import android.widget.ImageView
import com.fphoenixcorneae.bannerlayout.listener.BannerModelCallBack
import com.fphoenixcorneae.bannerlayout.listener.ImageDisplayManager
import com.fphoenixcorneae.bannerlayout.widget.BannerDefaults
import com.squareup.picasso.Picasso

class PicassoImageManager : ImageDisplayManager {
    override fun display(container: ViewGroup, model: BannerModelCallBack<*>?): ImageView {
        val imageView = ImageView(container.context)
        Picasso.get()
                .load(model?.bannerUrl as? String)
                .placeholder(BannerDefaults.GLIDE_PLACEHOLDER_DRAWABLE)
                .error(BannerDefaults.GLIDE_ERROR_DRAWABLE)
                .into(imageView)
        return imageView
    }
}